docs/api.md

```diff
@@ -211,6 +212,10 @@ to modify the result or reject the promise chain before it is sent to mixins.
 
 You retrieve a session by calling [`enigma.create(config)`](#enigmacreateconfig).
 
+### `session.config`
+
+A reference to this session's configuration object (including default values for optional parameters).
```

Comment:
this=a to keep the grammar consistent (I think we refer to "a session" in the other API sections)?

Reply:
Perhaps link the configuration text to the configuration section?